### DISCUSSION TITLE
Training Instructions for PointRend and DeepLab under Projects

### DISCUSSION THREAD

**Discussion by zhengjie6** [facebookresearch/detectron2#3045] [Q&A]
Hello,

I am trying to train with custom dataset of cityscape format (cityscape gtcoarse) and coco format (my own dataset) and I am wondering how to use those dataset for training with PointRend and DeepLab model that is parked under projects. Is there a tutorial on that?

Thank you in advance.

**Comment by Jaxi83** [comment]
I am also looking to train with custom datasets for semantic segmentation. 
I assume that I have t use cityscape format, but there is no tutorial on how to use it. 
Is there anybody that can help with this?
